pkg/codegen/go/gen_program.go

```diff
@@ -149,6 +151,11 @@ func GenerateProject(directory string, project workspace.Project, program *pcl.P
 
 	// Set the runtime to "go" then marshal to Pulumi.yaml
 	project.Runtime = workspace.NewProjectRuntimeInfo("go", nil)
+	if project.AdditionalKeys != nil {
```

Comment:
Bit lame having to do this project clean in each languages codegen, can't the convert code clean the project before passing it down to codegen?